storybook/stories/navigation/Sidebar.stories.js

```diff
@@ -59,9 +59,11 @@ const sidebarMenus = observable({
       },
     },
   },
+  hardwareWallets: null
```

Comment:
@DeeJayElly shouldn't it be an empty array?

Reply:
It's an empty object in this case.